Xamarin.Forms.Platform.Android/ContextExtensions.cs

```diff
@@ -96,5 +97,46 @@ public static AActivity GetActivity(this Context context)
 
 			return null;
 		}
+
+		public static Context GetThemedContext(this Context context)
+		{
+			if (context.IsDesignerContext())
+				return context;
+
+			if (context == null)
```

Comment:
You're null checking here after calling IsDesignerContext() on it. IsDesignerContext() does null check, but it would be neater to null check prior, for maintainability (someone could inadvertently follow suit with a non-extension method call after the first if block and trigger an NRE).